packages/itwinui-css/src/surface/surface.scss

```diff
@@ -22,5 +22,43 @@
 
 // Public facing surface component.
 @mixin iui-surface-component {
+  --iui-surface-padding-inline: var(--iui-size-s);
+  --iui-surface-body-padding: var(--iui-surface-padding-inline);
+
   @include iui-surface;
+  display: flex;
+  flex-direction: column;
```

Comment:
We cannot make this component flex or grid by default, because it could break apps that already use `<Surface>`. So it needs to go in a modifier class or attribute.